src/main/java/Duke.java

```diff
@@ -1,10 +1,336 @@
+            int indexOfTaskToBeMarked = taskNumber - 1;
+            if (indexOfTaskToBeMarked < taskList.size()) {
+                taskList.get(indexOfTaskToBeMarked).setDone(true);
+                String markedTaskAsDoneMessage =
+                        String.format("%s\t Nice! I've marked this task as done:\n \t %s %s \n%s",
+                                lineDivider,
+                                taskList.get(indexOfTaskToBeMarked).getStatusIcon(),
+                                taskList.get(indexOfTaskToBeMarked).getDescription(),
+                                lineDivider);
+                System.out.println(markedTaskAsDoneMessage);
+            } else {
+                String taskDoesNotExistMessage = String.format("%s\t Task does not exist!\n %s \n", lineDivider, lineDivider);
+                System.out.print(taskDoesNotExistMessage);
+            }
+        } else {
+            // empty list ! 
+            String doneErrorPrompt = "OOPS!!! Unable to mark task as done as your input is not in the right format!\n " +
+                    "Which task do you want to mark done?";
+            String doneListMessage = String.format("%s%s\n\t%s\n%s", lineDivider, doneErrorPrompt, getList(taskList), lineDivider);
+            System.out.println(doneListMessage);
+        }
+    }
+
+    public static void addTaskSuccessMessage(ArrayList<Task> taskList, String s) {
+        System.out.println(lineDivider);
+        System.out.println(s);
+        System.out.println("\t " + taskList.get(taskList.size() - 1).getStatusIcon()
+                + " " + taskList.get(taskList.size() - 1).getDescription());
+        System.out.println(lineDivider);
+    }
+
+    public static String getList(ArrayList<Task> taskList) {
+        if (!taskList.isEmpty()) { // userInput == list
+            StringBuilder sb = new StringBuilder();
+            String listAsString;
+            for (int i = 0; i < taskList.size(); i++) {
+                sb.append("\t\t");
+                sb.append((i + 1));
+                sb.append(". ");
+                sb.append(taskList.get(i).getStatusIcon());
+                sb.append(" ");
+                sb.append(taskList.get(i).getDescription());
+                sb.append("\n");
+                sb.toString();
+            }
+            sb.append("\n");
+            sb.append("\tNow you have ");
+            sb.append(taskList.size());
+            sb.append(" tasks in the list. \n");
+            sb.append("\tEnter \"done _\" to see mark task as done. \n");
+            sb.append("\tEnter \"delete _\" to see delete task. \n ");
+            listAsString = sb.toString();
+            return listAsString;
+        } else {
+            return "\tList is empty!\n";
+        }
+    }
+
+    public static void deleteTask(ArrayList<Task> taskList, String userInput) {
+        if (userInput.matches(".*\\d.*")) { // checks if there is a number in delete cmd
```

Comment:
I think comments should be indented relative to their position in the code. Refer to coding standards